go/test/endtoend/cluster/vtctlclient_process.go

```diff
@@ -73,25 +64,27 @@ func (vtctlclient *VtctlClientProcess) ApplySchema(Keyspace string, SQL string)
 
 // ApplyVSchema applies vitess schema (JSON format) to the keyspace
 func (vtctlclient *VtctlClientProcess) ApplyVSchema(Keyspace string, JSON string) (err error) {
-	file, err := ioutil.TempFile(vtctlclient.TempDirectory, "vschema.json")
-	if err != nil {
-		return
-	}
-	defer os.Remove(file.Name())
-
-	_, _ = file.WriteString(JSON)
-	_ = file.Close()
 	tmpProcess := exec.Command(
 		vtctlclient.Binary,
 		"-server", vtctlclient.Server,
 		"ApplyVSchema",
-		"-vschema_file", file.Name(),
+		"-vschema", JSON,
 		Keyspace,
 	)
 	print(fmt.Sprintf("ApplyVSchema with arguments %v", strings.Join(tmpProcess.Args, " ")))
 	return tmpProcess.Run()
 }
 
+// ExecuteCommand executes any vtctlclient command
+func (vtctlclient *VtctlClientProcess) ExecuteCommand(args ...string) (err error) {
+	tmpProcess := exec.Command(
+		vtctlclient.Binary,
+		args...,
+	)
+	print(fmt.Sprintf("ApplyVSchema with arguments %v", strings.Join(tmpProcess.Args, " ")))
```

Comment:
Change this log, it is not always ApplyVSchema.

Reply:
I already changed this in some earlier commit.